dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
+    login_type = (['none', '', nil].include? params[:login_type]) ? 'email' : params[:login_type]
+    return head :bad_request unless %w(word picture email).include? login_type
+
+    valid_script = params[:script] && valid_script_id?(params[:script][:id])
+    script_to_assign = valid_script && Script.find(params[:script][:id])
+
+    section = Section.create!(
```

Comment:
Might be better to do the non-bang create, and branch on `section.valid?` so we can `render head :bad_request` when it fails instead of blowing up.